### DISCUSSION TITLE
My account has been marked as spam.

### DISCUSSION THREAD

**Discussion by dev-hanzala** [community/community#145953] [New to GitHub]
While using https://cs50.dev/ my account got marked as spam with a banner saying, "You are marked as spam, and therefore cannot authorize a third-party application.". My friends can't see my profile or interact with my repos. This happened after an issue with the cs50 codespace which caused the codespace to not build correctly and get stuck into recovery mode. I followed this guide https://cs50.readthedocs.io/cs50.dev/#codespaces-run-into-recovery-mode which told me to create a new codespace. this worked for around 20 minutes but then the codespace suddenly stopped. when i tried to go back to https://cs50.dev/ it now redirects me to https://github.com/dashboard with that banner on top.

**Comment by ghostinhershell** [comment]
Hi @dev-hanzala! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them.
